<!doctype html public "-//w3c//dtd html 4.0 transitional//en">
<html>
<head>
   <meta http-equiv="Content-Type" content="text/html; charset=Shift_JIS">
   <meta name="Author" content="Asako Suzuki">
   <meta name="GENERATOR" content="Mozilla/4.6 [ja] (Win95; I) [Netscape]">
   <title>RECOMB2000 List of Accepted Papers</title>
</head>
<body text="#000000" bgcolor="#FFFFFF" link="#0000EE" vlink="#551A8B" alink="#FF0000">
<b><font size=+2>RECOMB 2000&nbsp; List
of accepted papers</font></b>
<p>Chor, Hendy, Holland, Penny
<br><i>Multiple Maxima of Likelihood in Phylogenetic Trees: An Analytic</i>
<br><i>Approach</i>
<br>&nbsp;
<p>Akutsu, Miyano, Kuhara
<br><i>Algorithms for Identifying Boolean Networks and Related Biological</i>
<br><i>Networks Based on Matrix Multiplication and Fingerprint Function</i>
<br>&nbsp;
<p>Marsan, Sagot
<br><i>Extracting structured motifs using a suffix tree - Algorithms and</i>
<br><i>application to promoter consensus identification</i>
<br>&nbsp;
<p>Linial, Portugaly
<br><i>Probabilities for having a new fold on the basis of a map of all</i>
<br><i>protein sequences</i>
<br>&nbsp;
<p>Chen, Kwong, Li
<br><i>A Compression Algorithm for DNA Sequences and Its Applications in
Genome</i>
<br><i>Comparison</i>
<br>&nbsp;
<p>Liwo, Pillardy, Czaplewski, Lee, Ripoll, Groth, Rodziewicz-Motowidlo,
<br>Kazmierkiewicz, Wawak, Oldziej,&nbsp; Scheraga
<br><i>UNRES -- a united-residue force field for energy-based prediction</i>
<br><i>of protein structure - origin and significance of multibody terms</i>
<br>&nbsp;
<p>Kann, Goldstein
<br><i>Optimizing for Success: A New Score Function For Distantly Related</i>
<br><i>Protein Sequence Comparison</i>
<br>&nbsp;
<p>Akutsu, Arimura, Shimozono
<br><i>On Approximation Algorithms for Local Multiple Alignment</i>
<br>&nbsp;
<p>Teichmann, Chothia, Church, Park
<br><i>PDB-ISL: an intermediate sequence library for fast assignment of</i>
<br><i>protein structures to sequences</i>
<br>&nbsp;
<p>Pevzner, Dancik, Tang
<br><i>Mutation-Tolerant Protein Identification by Mass-Spectrometry</i>
<br>&nbsp;
<p>Batzoglou, Berger, Mesirov, Lander
<br><i>Sequencing a Genome by Walking with Clone-end Sequences: A</i>
<br><i>Mathematical Analysis</i>
<br>&nbsp;
<p>Batzoglou, Pachter, Mesirov, Berger, Lander
<br><i>Comparative Analysis of Mouse and Human DNA and Applications to</i>
<br><i>Exon Prediction</i>
<br>&nbsp;
<p>Bienkowska, Yu, Zarakhovich, Rogers, Smith
<br><i>Comprehensive statistical method for protein fold recognition</i>
<br>&nbsp;
<p>Ohkubo, Crippen
<br><i>Determining Contact Energy Function for Continuous State Models</i>
<br><i>of Globular Protein Conformations</i>
<br>&nbsp;
<p>Bundschuh
<br><i>An Analytic Approach to Significance Assessment in Local Sequence</i>
<br><i>Alignment with Gaps</i>
<br>&nbsp;
<p>Apostolico, Bejerano
<br><i>Optimal Amnesic Probabilistic Automata or How to Learn and Classify</i>
<br><i>Proteins in Linear Time and Space</i>
<br>&nbsp;
<p>Heber, Stoye, Hoheisel, Vingron
<br><i>Contig Selection in Physical Mapping</i>
<br>&nbsp;
<p>Hallett, Lagergren
<br><i>New Algorithms for the Duplication-Loss Model</i>
<br>&nbsp;
<p>Desper, Vingron
<br><i>Tree Fitting: an Algebraic Approach Using Profile Distances</i>
<br>&nbsp;
<p>Lyngso, Pederson
<br><i>Pseudonknots in RNA Secondary Structures</i>
<br>&nbsp;
<p>Chen, Durand, Farach-Colton
<br><i>Notung: Dating gene duplications using gene family trees</i>
<br>&nbsp;
<p>Xu, Xu, Crawford, Einstein, Serpersu
<br><i>Protein Structure Determination using Protein Threading and</i>
<br><i>Sparse&nbsp; NMR Data</i>
<br>&nbsp;
<p>Slonim, Tamayo, Mesirov, Golub, Lander
<br><i>Class prediction and discovery using gene expression data</i>
<br>&nbsp;
<p>Sorenson, Head-Gordon
<br><i>Matching simulation and experiment: a new simplified model for</i>
<br><i>simulating protein folding</i>
<br>&nbsp;
<p>Zien, Zimmer, Lengauer
<br><i>A simple iterative approach to parameter optimization</i>
<br>&nbsp;
<p>Ben-Dor, Karp, Schwikowski, Yakhini
<br><i>Universal DNA tag systems: a combinatorial design scheme</i>
<br>&nbsp;
<p>Sankoff, Bryant, Deneault, Lang, Burger
<br><i>Early Eukaryote Evolution Based on Mitochondrial Gene Order</i>
<br><i>Breakpoints</i>
<br>&nbsp;
<p>Yona, Levitt
<br><i>A unified sequence-structure classification of protein sequences:</i>
<br><i>combining sequence and structure in a map of the protein space</i>
<br>&nbsp;
<p>Bailey-Kellogg, Widge, Kelley, Berardi, Bushweller, Donald
<br><i>The NOESY Jigsaw: automated protein secondary structure and</i>
<br><i>main-chain assignment from sparse, unassigned NMR data</i>
<br>&nbsp;
<p>Ben-Dor, Bruhn, Friedman, Nachman, Schummer, Yakhini
<br><i>Tissue classification with gene expression profiles</i>
<br>&nbsp;
<p>Kececioglou, Shete, Arnold
<br><i>Reconstructing distances in physical maps of chromosomes with</i>
<br><i>nonoverlapping probes</i>
<br>&nbsp;
<p>Hart, Royyuru, Stolovitzky, Califano
<br><i>Systematic and Automated Discovery of Patterns in PROSITE Families</i>
<br>&nbsp;
<p>Althaus, Kohlbacher, Lenhof, Mueller
<br><i>A combinatorial approach to protein docking with flexible side-chains</i>
<br>&nbsp;
<p>Friedman, Linial, Nachman, Pe'er
<br><i>Using Bayesian Networks to Analyze Expression Data</i>
<br>&nbsp;
<p>Preparata, Upfal
<br><i>Sequencing-by-hybridization at the information-theory bound:</i>
<br><i>an optimal algorithm</i>
<br>&nbsp;
<p>Tomita, Hashimoto, Takahashi, Matsuzaki, Matsushima, Yugi, Miyoshi,
<br>Nakano, Saito, Shimizu, Nakayama
<br><i>The E-CELL project: towards integrative simulation of cellular</i>
<br><i>processes</i>
<br>&nbsp;
<br>&nbsp;
<br>&nbsp;
</body>
</html>



