<html>
<head>
   <title>RECOMB 2000 List of accepted posters</title>
</head>
<body bgcolor="#FFFFFF">

<H2>List of Accepted Posters</H2>

<OL>

<li>
<B>Comparative Study of Clustering Techniques for Gene Expression Microarray Data</B>
<br>--- Sonia Leach, Larry Hunter
<br>[ <A HREF="pdf/1.pdf">PDF</A> ] [ <A HREF="ps/1.ps.gz">PS</A> ]

<P><li>
<B>A Computer Program for Generating Gene-Specific Fragments for Microarrays</B>
<br>--- Dong Xu, Ying Xu, Gary Li, Jizhong Zhou
<br>[ <A HREF="pdf/2.pdf">PDF</A> ] [ <A HREF="ps/2.ps.gz">PS</A> ]

<P><li>
<B>Approaching Causality: Discovering Time-Lag Correlations in Genetic Expression Data with Static and Dynamic Relevance Networks</B>
<br>--- Ben Y. Reis, Atul J. Butte, Isaac S. Kohane
<br>[ <A HREF="pdf/3.pdf">PDF</A> ] [ <A HREF="ps/3.ps.gz">PS</A> ]

<P><li>
<B>CLICK: A Clustering Algorithm for Gene Expression Analysis</B>
<br>--- Ron Shamir, Roded Sharan
<br>[ <A HREF="pdf/4.pdf">PDF</A> ] [ <A HREF="ps/4.ps.gz">PS</A> ]

<P><li>
<B>Inference of Nonlinear Biological Systems by Using Linear Programming</B>
<br>--- Tatsuya Akutsu, Satoru Miyano, Satoru Kuhara
<br>[ <A HREF="pdf/5.pdf">PDF</A> ] [ <A HREF="ps/5.ps.gz">PS</A> ]

<P><li>
<B>Template-Based Gene Expression Analysis</B>
<br>--- Torgeir R. Hvidsten, Tor-Kristian Jenssen, Jan Komorowski, Astrid L&#230;greid, Arne Sandvik, Dyre Tjeldvoll
<br>[ <A HREF="pdf/6.pdf">PDF</A> ] [ <A HREF="ps/6.ps.gz">PS</A> ]

<P><li>
<B>Gene Expression Analysis by DNA Computing</B>
<br>--- Akira Suyama, Nao Nishida, Ken-ichi Kurata, Katsumi Omagari
<br>[ <A HREF="pdf/7.pdf">PDF</A> ] [ <A HREF="ps/7.ps.gz">PS</A> ]

<P><li>
<B>Ordering Genes by Significance of Occurrences</B>
<br>--- Jun Sese, Shinichi Morishita, Kousaku Okubo
<br>[ <A HREF="pdf/8.pdf">PDF</A> ] [ <A HREF="ps/8.ps.gz">PS</A> ]

<P><li>
<B>Diagnosis System of Drug Sensitivity of Cancer Using cDNA Microarray and Multivariate Statistical Analysis</B>
<br>--- Tatsuhiko Tsunoda, Yoshiaki Hojo, Chikashi Kihara, Norihiko Shiraishi, Osamu Kitahara, Kenji Ono, Toshihiro Tanaka, Toshihisa Takagi, Yusuke Nakamura
<br>[ <A HREF="pdf/9.pdf">PDF</A> ] [ <A HREF="ps/9.ps.gz">PS</A> ]

<P><li>
<B>An Evaluation of Single Nucleotide Polymorphism Detection Programs</B>
<br>--- Rosalind Cutts, Ravi Mehta, Suzanne Orton, Philip East, Robert Gill, Philippe Sanseau, Paul Mathews
<br>[ <A HREF="pdf/10.pdf">PDF</A> ] [ <A HREF="ps/10.ps.gz">PS</A> ]

<P><li>
<B>Arrayed Primer Extension on DNA Microchips (APEX). Molecular Computation
of Satisfaction (SAT) Problems</B>
<br>--- Micheal Pirrung, Richard Connors, Amy Odenbaugh, Michael Montague-Smith, Nathan Walcott, Jeff Tollett
<br>[ <A HREF="pdf/11.pdf">PDF</A> ] [ <A HREF="ps/11.ps.gz">PS</A> ]

<P><li>
<B>ViewDesigner: A Tool of Designing Views on Data in Discovery System Hypothesis Creator</B>
<br>--- Osamu Maruyama, Yoshinori Tamada, Satoshi Matsumoto, Satoru Miyano
<br>[ <A HREF="pdf/12.pdf">PDF</A> ] [ <A HREF="ps/12.ps.gz">PS</A> ]

<P><li>
<B>A System to Find Genetic Networks Based on Weighted Majority Algorithm</B>
<br>--- Ayumi Shinohara, Masayuki Takeda, Tomohiro Moriyama, Osamu Maruyama, Takao Goto, Satoru Miyano, Shigeru Muta, Koushuke Tashiro, Satoru Kuhara
<br>[ <A HREF="pdf/13.pdf">PDF</A> ] [ <A HREF="ps/13.ps.gz">PS</A> ]

<P><li>
<B>Induction Mechanism Description of <font face=symbol>&#108;</font> Phage by Hybrid Petri Net</B>
<br>--- Atsushi Doi, Hiroshi Matsuno, Satoru Miyano
<br>[ <A HREF="pdf/14.pdf">PDF</A> ] [ <A HREF="ps/14.ps.gz">PS</A> ]

<P><li>
<B>Bio-calculus: Its Concept, and An Application for Molecular Interaction</B>
<br>--- Masao Nagasaki, Shuichi Onami, Satoru Miyano, Hiroaki Kitano
<br>[ <A HREF="pdf/15.pdf">PDF</A> ] [ <A HREF="ps/15.ps.gz">PS</A> ]

<P><li>
<B>Software Platform for Systems Biology</B>
<br>--- Adam Arkin, Hamid Bolouri, Mark T. Borisuk, John Doyle, Koji, M.Kyoda, Eric Mjolsness, Mineo Morohashi, Hiroaki Kitano
<br>[ <A HREF="pdf/16.pdf">PDF</A> ] [ <A HREF="ps/16.ps.gz">PS</A> ]

<P><li>
<B>Reverse Engineering in Systems Biology: Towards Inference of a Gene Regulatory Network</B>
<br>--- Mineo Morohashi, Koji M. Kyoda, Shuichi Onami, Eric Mjolsness, Hiroaki Kitano
<br>[ <A HREF="pdf/17.pdf">PDF</A> ] [ <A HREF="ps/17.ps.gz">PS</A> ]

<P><li>
<B>Development of a System for Automatic Construction of Cell Lineage of <I>C. elegans</I> from Nomarski DIC Microscope Images</B>
<br>--- Tomohiro Yasuda, Hideo Bannai, Shuichi Onami, Satoru Miyano, Hiroaki Kitano
<br>[ <A HREF="pdf/18.pdf">PDF</A> ] [ <A HREF="ps/18.ps.gz">PS</A> ]

<P><li>
<B>Systematic Analysis of the Robustness in Complex Reaction Networks of Bacteria</B>
<br>--- Hiroyuki Kurata, Kazunari Taira
<br>[ <A HREF="pdf/19.pdf">PDF</A> ] [ <A HREF="ps/19.ps.gz">PS</A> ]

<P><li>
<B>Genius: An Automated System for Assigning Genome ORFs to 3D Structures, Using a Novel Multiple Intermediate Sequence Search</B>
<br>--- Makiko Suwa, Henrik T. Yudate, Yasuhiko Masuho, Asaf A. Salamov, Christine A. Orengo, Mark B. Swindells
<br>[ <A HREF="pdf/20.pdf">PDF</A> ] [ <A HREF="ps/20.ps.gz">PS</A> ]

<P><li>
<B>High-Throughput Function Assignment for Novel Gene Products using Annotation Clustering</B>
<br>--- Alexander Renner, Hilmar Lapp, Andras Aszodi
<br>[ <A HREF="pdf/21.pdf">PDF</A> ] [ <A HREF="ps/21.ps.gz">PS</A> ]

<P><li>
<B>CANVAS: A Clone Annotation and Validation System</B>
<br>--- Zhong Li, Junping Jing, Jeffrey S. Aaronson
<br>[ <A HREF="pdf/22.pdf">PDF</A> ] [ <A HREF="ps/22.ps.gz">PS</A> ]

<P><li>
<B>A 1.7 Mb Sequence Analysis of Rice Chromosome 1</B>
<br>--- Atsuko Idonuma, Hideki Nagasaki, Masatoshi Masukawa, Manami Negishi, Koji Arikawa, Yoshiyuki Mukai, Isamu Ohta, Baltazar A. Antonio, Katsumi Sakata, Takuji Sasaki
<br>[ <A HREF="pdf/23.pdf">PDF</A> ] [ <A HREF="ps/23.ps.gz">PS</A> ]

<P><li>
<B>Web-Based System for Sample Tracking in Rice Genome Research Program</B>
<br>--- Kazunori Waki, Kimiko Yamamoto, Isamu Ohta, Masaki Kise, Katsumi Sakata, Takuji Sasaki
<br>[ <A HREF="pdf/24.pdf">PDF</A> ] [ <A HREF="ps/24.ps.gz">PS</A> ]

<P><li>
<B>Pubgen: Discovering and Visualising Gene-Gene Relations</B>
<br>--- Tor-Kristian Jenssen, Astrid Lagreid, Jan Komorowski, Evind Hovig
<br>[ <A HREF="pdf/25.pdf">PDF</A> ] [ <A HREF="ps/25.ps.gz">PS</A> ]

<P><li>
<B>Genome Analysis and Gene Discovery Tools</B>
<br>--- Christian Iseli, C.Victor Jongeneel, Marco Pagni, Thomas Junier, Philipp Bucher
<br>[ <A HREF="pdf/26.pdf">PDF</A> ] [ <A HREF="ps/26.ps.gz">PS</A> ]

<P><li>
<B>Gene Discovery and Presentation Systems for <I>Arabidopsis</I> Genome Sequencing Project at Kazusa DNA Research Institute</B>
<br>--- Yasukazu Nakamura, Satoshi Tabata
<br>[ <A HREF="pdf/27.pdf">PDF</A> ] [ <A HREF="ps/27.ps.gz">PS</A> ]

<P><li>
<B>GeneHacker Plus: Yet Another Gene-Finding Program with Top-Class Accuracy in Locating Start Codons</B>
<br>--- Tetsushi Yada, Yasushi Totoki, Toshihisa Takagi, Kenta Nakai
<br>[ <A HREF="pdf/28.pdf">PDF</A> ] [ <A HREF="ps/28.ps.gz">PS</A> ]

<P><li>
<B>Predicting Gene Boundary</B>
<br>--- Sridhar S. Hannenhalli, James W. Fickett
<br>[ <A HREF="pdf/29.pdf">PDF</A> ] [ <A HREF="ps/29.ps.gz">PS</A> ]

<P><li>
<B>A Workbench for the Research and Development of Biological Diversity</B>
<br>--- Hideaki Sugawara, Satoru Miyazaki
<br>[ <A HREF="pdf/30.pdf">PDF</A> ] [ <A HREF="ps/30.ps.gz">PS</A> ]

<P><li>
<B>A New File Format and Tools for the Large-Scale Data Submission to DNA Data Bank of Japan(DDBJ)</B>
<br>--- Satoru Miyazaki, Hiroyuki Hashimoto, Akemi Shimada, Yoshio Tateno, Hideaki Sugawara
<br>[ <A HREF="pdf/31.pdf">PDF</A> ] [ <A HREF="ps/31.ps.gz">PS</A> ]

<P><li>
<B>A Proposal for Integration of BLAST-Related Software Tools Using CORBA</B>
<br>--- Pethuru Raj, Naohiro Ishii
<br>[ <A HREF="pdf/32.pdf">PDF</A> ] [ <A HREF="ps/32.ps.gz">PS</A> ]

<P><li>
<B>GeneNet: A Meta-Sequence Similarity Analysis System</B>
<br>--- Dong-Sun Park, Sung Hwan Kim, Tae-Jin Ahn, Yang-Suk Kim, Hong-gil Nam
<br>[ <A HREF="pdf/33.pdf">PDF</A> ] [ <A HREF="ps/33.ps.gz">PS</A> ]

<P><li>
<B>Visualysis: A Tool for Biological Sequence Analysis</B>
<br>--- SungHwan Kim, YangSuk Kim, TaeJin Ahn, Hong Kil Nam, Byung Jun Han, Sam Myo Kim
<br>[ <A HREF="pdf/34.pdf">PDF</A> ] [ <A HREF="ps/34.ps.gz">PS</A> ]

<P><li>
<B>GeneAtlas<SUP>TM</SUP>: A High-Throughput Pipeline for Automated Model Building and Functional Annotation of the Genome</B>
<br>--- David Edwards, Zhan-yang Zhu, Azat Badretdinov, David Kitson, Krzysztof Olszewski, Lisa Yan
<br>[ <A HREF="pdf/36.pdf">PDF</A> ] [ <A HREF="ps/36.ps.gz">PS</A> ]

<P><li>
<B>GeneAtlas<SUP>TM</SUP> - An Automatic High-Throughput Pipeline for Structure Prediction and Function Assignment for Genomic Sequences</B>
<br>--- Lisa Yan, Zhan-Yang Zhu, Azat Badredinov, David Kitson, Krzysztof Olszewski, David Edwards
<br>[ <A HREF="pdf/35.pdf">PDF</A> ] [ <A HREF="ps/35.ps.gz">PS</A> ]

<P><li>
<B>The 2D Gel Electrophoresis Simulator: <I>SimGel</I></B>
<br>--- Chungfan Kim, Akihiko Konagaya
<br>[ <A HREF="pdf/37.pdf">PDF</A> ] [ <A HREF="ps/37.ps.gz">PS</A> ]

<P><li>
<B>EstSearch: A Tool for More Informative EST Alignments</B>
<br>--- Vera Asodi, Alex Diber, Raveh Gill-More, Hershel Safer, Mor Amitai
<br>[ <A HREF="pdf/38.pdf">PDF</A> ] [ <A HREF="ps/38.ps.gz">PS</A> ]

<P><li>
<B>Clustering and Assmbly of a Large Number of EST and cDNA Sequences Using Hyseq's Proprietary Software</B>
<br>--- John Tillinghast, Ankura Sinku, Y. Tom Tang
<br>[ <A HREF="pdf/39.pdf">PDF</A> ] [ <A HREF="ps/39.ps.gz">PS</A> ]

<P><li>
<B>Can General-Purpose Compression Schemes Really Compress DNA Sequences?</B>
<br>--- Toshiko Matsumoto, Kunihiko Sadakane, Hiroshi Imai, Takumi Okazaki
<br>[ <A HREF="pdf/40.pdf">PDF</A> ] [ <A HREF="ps/40.ps.gz">PS</A> ]

<P><li>
<B>The Eukaryotic Promoter Database</B>
<br>--- Rouaida Cavin Perier, Viviane Praz, Thomas Junier, Claude Bonnard, Christian Iseli, Philipp Bucher
<br>[ <A HREF="pdf/41.pdf">PDF</A> ] [ <A HREF="ps/41.ps.gz">PS</A> ]

<P><li>
<B>CAMUS DB: Development of Structural Database for Homology Search</B>
<br>--- Masayuki Kikuchi, Sadahiko Misu, Tadashi Imanishi, Naruya Saitou
<br>[ <A HREF="pdf/42.pdf">PDF</A> ] [ <A HREF="ps/42.ps.gz">PS</A> ]

<P><li>
<B>Upgraded Aberrant Splicing Database Supports the Scanning Mechanism of 3' Splice-Site Selection</B>
<br>--- Takashi Yamanaka, Tetsushi Yada, Toshihisa Takagi, Kenta Nakai
<br>[ <A HREF="pdf/43.pdf">PDF</A> ] [ <A HREF="ps/43.ps.gz">PS</A> ]

<P><li>
<B>Receptor Database (RDB) As an Analytical Tool</B>
<br>--- Kotoko Nakata, Takako Takai, Tatsuya Nakano, Tsuguchika Kaminuma
<br>[ <A HREF="pdf/44.pdf">PDF</A> ] [ <A HREF="ps/44.ps.gz">PS</A> ]

<P><li>
<B>Transcript Sequence Registry</B>
<br>--- Junping Jing, Jian Jiang, Jeffrey S. Aaronson
<br>[ <A HREF="pdf/45.pdf">PDF</A> ] [ <A HREF="ps/45.ps.gz">PS</A> ]

<P><li>
<B>Building Integrated Pathway Genome Database in Cell Type Specific Manner - Encyclopedia of Human Liver</B>
<br>--- Taejin Ahn, YangSuk Kim, SungHwan Kim, DongSun Park, HongGil Nam
<br>[ <A HREF="pdf/46.pdf">PDF</A> ] [ <A HREF="ps/46.ps.gz">PS</A> ]

<P><li>
<B>A Structured Genome Document Database System for Making High-Level Queries on Diverse Genome Data</B>
<br>--- Aaron Stokes, Hideo Matsuda, Akihiko Hashimoto
<br>[ <A HREF="pdf/47.pdf">PDF</A> ] [ <A HREF="ps/47.ps.gz">PS</A> ]

<P><li>
<B>SST: An Algorithm for Searching Sequence Databases in Time Proportional to the Logarithm of the Database Size</B>
<br>--- Eldar Giladi, Michael G. Walker, James Ze Wang, Wayne Volkmuth
<br>[ <A HREF="pdf/48.pdf">PDF</A> ] [ <A HREF="ps/48.ps.gz">PS</A> ]

<P><li>
<B>Design of the Comprehensive Fold Recognition Benchmark. Application to SeqFold, Training and Validation.</B>
<br>--- Krzysztof A. Olszewski
<br>[ <A HREF="pdf/49.pdf">PDF</A> ] [ <A HREF="ps/49.ps.gz">PS</A> ]

<P><li>
<B>Isothermic Oligonucleotide Libraries</B>
<br>--- Jacek Blazewicz, Piotr Formanowicz, Marta Kasprzak, Wojciech T.Markiewicz
<br>[ <A HREF="pdf/50.pdf">PDF</A> ] [ <A HREF="ps/50.ps.gz">PS</A> ]

<P><li>
<B>Estimation of Stochastic Parameters</B>
<br>--- Michael A. Gibson, Jehoshua Bruck
<br>[ <A HREF="pdf/51.pdf">PDF</A> ] [ <A HREF="ps/51.ps.gz">PS</A> ]

<P><li>
<B>Computational Prediction of Gene Regulation by DNA Conformation</B>
<br>--- Mick Noordewier, Peter Morrison, Craig Benham
<br>[ <A HREF="pdf/52.pdf">PDF</A> ] [ <A HREF="ps/52.ps.gz">PS</A> ]

<P><li>
<B>Method for Recognition of Composite Regulatory Units in Promoters. Analysis of Cell-Cycle Control Genes</B>
<br>--- Alexander Kel, Olga Kel-Margoulis, Edgar Wingender
<br>[ <A HREF="pdf/53.pdf">PDF</A> ] [ <A HREF="ps/53.ps.gz">PS</A> ]

<P><li>
<B>Algorithm Determining All Cut-off Values of TF-DNA Binding Score Calculated Using PWMs in TRANSFAC</B>
<br>--- Tatsuhiko Tsunoda, Toshihisa Takagi
<br>[ <A HREF="pdf/54.pdf">PDF</A> ] [ <A HREF="ps/54.ps.gz">PS</A> ]

<P><li>
<B>Structure-Based Target Prediction of Transcription Factors</B>
<br>--- Akinori Sarai, Hidetoshi Kono
<br>[ <A HREF="pdf/55.pdf">PDF</A> ] [ <A HREF="ps/55.ps.gz">PS</A> ]

<P><li>
<B>An Approach to Alternative Gene Transcripts Discovery through Assembly of Fragmental cDNA Sequences</B>
<br>--- Ryotaro Irie, Yasuhiko Masuho, Keiichi Nagai
<br>[ <A HREF="pdf/56.pdf">PDF</A> ] [ <A HREF="ps/56.ps.gz">PS</A> ]

<P><li>
<B>Association of Binding Specificity and Disordered Regions in Zinc Finger Motifs</B>
<br>--- Takeshi Nagashima, Akihiko Konagaya
<br>[ <A HREF="pdf/57.pdf">PDF</A> ] [ <A HREF="ps/57.ps.gz">PS</A> ]

<P><li>
<B>A New Method for Matching Protein Secondary Structure Elements or C-alpha Atoms in Protein Structure Comparison</B>
<br>--- Edward S.C. Shih, Tsan-Fu Wang, Ming-Jing Hwang
<br>[ <A HREF="pdf/58.pdf">PDF</A> ] [ <A HREF="ps/58.ps.gz">PS</A> ]

<P><li>
<B>Sequence Codes for Extended Conformation: A Neighbor-Dependent Sequence Analysis of Loops in Proteins</B>
<br>--- Jin-an Feng, Chiquito J. Crasto
<br>[ <A HREF="pdf/59.pdf">PDF</A> ] [ <A HREF="ps/59.ps.gz">PS</A> ]

<P><li>
<B>Extending HP Lattice Model with Non-Local Hydration</B>
<br>--- Jarmo T. Alander
<br>[ <A HREF="pdf/60.pdf">PDF</A> ] [ <A HREF="ps/60.ps.gz">PS</A> ]

<P><li>
<B>A New Method for Evaluating the Structural Similarity of Proteins Using Geometric Morphometrics</B>
<br>--- Dean C. Adams, Gavin J.P. Naylor
<br>[ <A HREF="pdf/61.pdf">PDF</A> ] [ <A HREF="ps/61.ps.gz">PS</A> ]

<P><li>
<B>A Methodology to Optimally Extract Local Structural Information in Proteins</B>
<br>--- Armando D. Solis, S. Rackovsky
<br>[ <A HREF="pdf/62.pdf">PDF</A> ] [ <A HREF="ps/62.ps.gz">PS</A> ]

<P><li>
<B>Optimal Hidden Markov Models for All Sequences of Known Structure</B>
<br>--- Julian Gough, Cyrus Chothia, Kevin Karplus, Christian Barrett, Richard Hughey
<br>[ <A HREF="pdf/63.pdf">PDF</A> ] [ <A HREF="ps/63.ps.gz">PS</A> ]

<P><li>
<B>IMGT Sequence Profile: A Standardized Visualization for the Immunoglobulin and T Cell Receptor V-REGIONs Applicable to Other Protein Alignments</B>
<br>--- Manule Ruiz, Marie-Paule Lefranc
<br>[ <A HREF="pdf/64.pdf">PDF</A> ] [ <A HREF="ps/64.ps.gz">PS</A> ]

<P><li>
<B>Motif Recognition in Genomes</B>
<br>--- Peter Clote, Sebastian Will
<br>[ <A HREF="pdf/65.pdf">PDF</A> ] [ <A HREF="ps/65.ps.gz">PS</A> ]

<P><li>
<B>Analysis of Antisense Oligodeoxynucleotides Based on Sequence Motif Content to Predict Effectiveness</B>
<br>--- Michael Giddings, Olga Matveeva, Alex Tsodikov, Raymond F. Gesteland, John F. Atkins
<br>[ <A HREF="pdf/66.pdf">PDF</A> ] [ <A HREF="ps/66.ps.gz">PS</A> ]

<P><li>
<B>Comparison of Profile and Hidden Markov Model Methods for Detecting Degenerate Motifs in Protein Sequences</B>
<br>--- Krzysztof M. Rajkowski, Etienne-Emile Baulieu, Michael Schumacher
<br>[ <A HREF="pdf/67.pdf">PDF</A> ] [ <A HREF="ps/67.ps.gz">PS</A> ]

<P><li>
<B>Improving the Accuracy of Functional Inferences from Links between Proteins
Detected by Computational or Experimental Methods</B>
<br>--- Michael Thompson, Edward Marcotte, Matteo Pellegrini, Todd Yeates, David Eisenberg
<br>[ <A HREF="pdf/68.pdf">PDF</A> ] [ <A HREF="ps/68.ps.gz">PS</A> ]

<P><li>
<B>Evaluating Homology Search Algorithms</B>
<br>--- Timothy L. Bailey, Michael Gribskov
<br>[ <A HREF="pdf/69.pdf">PDF</A> ] [ <A HREF="ps/69.ps.gz">PS</A> ]

<P><li>
<B>Protein Superfamily Building Methods Comparison</B>
<br>--- Kunbin Qu, Jun Zhu, Scott Pegg, Patricia Babbitt
<br>[ <A HREF="pdf/70.pdf">PDF</A> ] [ <A HREF="ps/70.ps.gz">PS</A> ]

<P><li>
<B>Clustering Protein Sequences: Structure Prediction by Transitive Homology</B>
<br>--- Eva Bolten, Alexander Schliep, Sebastian Schneckener, D. Schomburg, Rainer Schrader
<br>[ <A HREF="pdf/71.pdf">PDF</A> ] [ <A HREF="ps/71.ps.gz">PS</A> ]

<P><li>
<B>Side-Chain Clusters in Protein Structures</B>
<br>--- N. Kannan, S. Vishveshwara
<br>[ <A HREF="pdf/72.pdf">PDF</A> ] [ <A HREF="ps/72.ps.gz">PS</A> ]

<P><li>
<B>Hierarchical Clustering Procedure for Grouping Orthologous Domains in Multiple Genomes</B>
<br>--- Ikuo Uchiyama
<br>[ <A HREF="pdf/73.pdf">PDF</A> ] [ <A HREF="ps/73.ps.gz">PS</A> ]

<P><li>
<B>The Grammatical Structure of Protein Sequences in a Single Non-linear Hidden Markov Model; Prediction of Coding Regions, Secondary and Tertiary structure</B>
<br>--- Christopher Bystroff
<br>[ <A HREF="pdf/74.pdf">PDF</A> ] [ <A HREF="ps/74.ps.gz">PS</A> ]

<P><li>
<B>The Statistics of Semi-Probabilistic Alignment</B>
<br>--- Yi-Kuo Yu, Terence Hwa
<br>[ <A HREF="pdf/75.pdf">PDF</A> ] [ <A HREF="ps/75.ps.gz">PS</A> ]

<P><li>
<B>Topological Requirement for the Nucleus Formation of a Two-state Folding Reaction. First Successful Blind <font face=symbol>&#70;</font>-Values Predictions - Activation Domain of Human Procarboxypeptidase</B>
<br>--- Roumen A. Dimitrov
<br>[ <A HREF="pdf/76.pdf">PDF</A> ] [ <A HREF="ps/76.ps.gz">PS</A> ]

<P><li>
<B>Conformer Models of TEM-1 <font face=symbol>&#98;</font>-lactamase</B>
<br>--- Christo Christov, Boris Atanasov
<br>[ <A HREF="pdf/77.pdf">PDF</A> ] [ <A HREF="ps/77.ps.gz">PS</A> ]

<P><li>
<B>The Identification of Transmembrane Helices in the Sequences of Membrane
Proteins Using a Computationally-Derived Hydrophobicity Scale</B>
<br>--- Lisa Bourla, Tidhar Seifer, Barry Honig, Nir Ben-Tal
<br>[ <A HREF="pdf/78.pdf">PDF</A> ] [ <A HREF="ps/78.ps.gz">PS</A> ]

<P><li>
<B>Prediction of Hydrophobic Cores of Globular Proteins Using Multiresolution Analysis of Wavelet Analysis</B>
<br>--- Hideki Hirakawa, Satoru Kuhara
<br>[ <A HREF="pdf/79.pdf">PDF</A> ] [ <A HREF="ps/79.ps.gz">PS</A> ]

<P><li>
<B>Structure Prediction of Proteins with Transmembrane Helices</B>
<br>--- Takatsugu Hirokawa, Makiko Suwa, Seah Boon-Chieng, Shigeki Mitaku
<br>[ <A HREF="pdf/80.pdf">PDF</A> ] [ <A HREF="ps/80.ps.gz">PS</A> ]

<P><li>
<B>Creating Physical Maps With Automatic Caputure of Hybridization Data</B>
<br>--- David Hall, Suchendra Bhandarkar, Jonathan Arnold
<br>[ <A HREF="pdf/81.pdf">PDF</A> ] [ <A HREF="ps/81.ps.gz">PS</A> ]

<P><li>
<B>Probe Interval Graph and Its Applications to Physical Mapping of DNA</B>
<br>--- Peisen Zhang
<br>[ <A HREF="pdf/82.pdf">PDF</A> ] [ <A HREF="ps/82.ps.gz">PS</A> ]

<P><li>
<B>Using Genetic Algorithms to Construct Physical Maps of Chromosomes with Unique Probes</B>
<br>--- Huai-Kuang Tsai, Cheng-Yan Kao
<br>[ <A HREF="pdf/83.pdf">PDF</A> ] [ <A HREF="ps/83.ps.gz">PS</A> ]

<P><li>
<B>Graph Modeling of Metabolism</B>
<br>--- Masanori Arita, Kiyoshi Asai, Takaaki Nishioka
<br>[ <A HREF="pdf/84.pdf">PDF</A> ] [ <A HREF="ps/84.ps.gz">PS</A> ]

<P><li>
<B>A Local Alignment Algorithm for Metabolic Pathway Analysis</B>
<br>--- Yukako Tohsato, Hideo Matsuda, Akihiro Hashimoto
<br>[ <A HREF="pdf/85.pdf">PDF</A> ] [ <A HREF="ps/85.ps.gz">PS</A> ]

<P><li>
<B>Optimizing RNA Secondary Structure Over All Possible Encodings of a Given Protein</B>
<br>--- Barry Cohen, Steven Skiena
<br>[ <A HREF="pdf/86.pdf">PDF</A> ] [ <A HREF="ps/86.ps.gz">PS</A> ]

<P><li>
<B>PiKA<SUP>2</SUP>: A WWW Based System for Automated Analysis of Protein 2D-PAGE Images</B>
<br>--- Katsutoshi Takahashi, Masayuki Nakazawa, Yasuo Watanabe, Akihiko Konagaya
<br>[ <A HREF="pdf/87.pdf">PDF</A> ] [ <A HREF="ps/87.ps.gz">PS</A> ]

<P><li>
<B>Improved Elastic Rod Model of 3D RNA Structure Formation</B>
<br>--- Eugeny Kugushev, Ekaterina Kozyreva, Aleksey Maykov
<br>[ <A HREF="pdf/88.pdf">PDF</A> ] [ <A HREF="ps/88.ps.gz">PS</A> ]

<P><li>
<B>Phylogenetic Reconstruction of Influenza B from 1940 to 1998. Are Means Important For The End?</B>
<br>--- L.M. Boykin, H. Lu, C.A. Macken, P.H. Mukundan, A.K. Naik
<br>[ <A HREF="pdf/89.pdf">PDF</A> ] [ <A HREF="ps/89.ps.gz">PS</A> ]

<P><li>
<B>Comparative Analysis of Genomic Rearrangements in Microorganisms</B>
<br>--- Siv G E Andersson, Daniel Dalevi, Niklas Eriksen, Kimmo Eriksson
<br>[ <A HREF="pdf/90.pdf">PDF</A> ] [ <A HREF="ps/90.ps.gz">PS</A> ]

<P><li>
<B>Maximum Likelihood Reconstruction of Ancestral Amino-acid Sequences</B>
<br>--- Tal Pupko, Itsik Pe'er
<br>[ <A HREF="pdf/91.pdf">PDF</A> ] [ <A HREF="ps/91.ps.gz">PS</A> ]

<P><li>
<B>Molecular Phylogenetics of the Genus <I>Oryza</I> L. (Poaceae): Evidence from the Sequences of the Internal Transcribed Spacer (ITS) of Nuclear Ribosomal DNA</B>
<br>--- Xie Zhongwen, Zhou Yi, Lu Baorong, Hong Deyuan
<br>[ <A HREF="pdf/92.pdf">PDF</A> ] [ <A HREF="ps/92.ps.gz">PS</A> ]

<P><li>
<B>Incomplete Directed Perfect Phylogeny</B>
<br>--- Itsik Pe'er, Ron Shamir, Roded Sharan
<br>[ <A HREF="pdf/93.pdf">PDF</A> ] [ <A HREF="ps/93.ps.gz">PS</A> ]

<P><li>
<B>Comparison of Information Criteria and Simulation Statistics Method for Topology Selection in Molecular Phylogenetic Tree</B>
<br>--- Soichi Ogishima, Fengrong Ren, Hiroshi Tanaka
<br>[ <A HREF="pdf/94.pdf">PDF</A> ] [ <A HREF="ps/94.ps.gz">PS</A> ]

<P><li>
<B>Inferring a Species Tree from Gene Trees under the Deep Coalescence Cost</B>
<br>--- Louxin Zhang
<br>[ <A HREF="pdf/95.pdf">PDF</A> ] [ <A HREF="ps/95.ps.gz">PS</A> ]

<P><li>
<B>Optimizing Pedigree Drawing Using Interval Graph Theory</B>
<br>--- Frederic Tores, Emmanuel Barillot
<br>[ <A HREF="pdf/96.pdf">PDF</A> ] [ <A HREF="ps/96.ps.gz">PS</A> ]

<P><li>
<B>Similarity Trees for Zinc Finger Protein Design</B>
<br>--- Dagmar Steffen, Dusan Ihracky, Lothar Gierl
<br>[ <A HREF="pdf/97.pdf">PDF</A> ] [ <A HREF="ps/97.ps.gz">PS</A> ]

<P><li>
<B>Singular Value Decomposition of Protein Conformational Motions: Application to HIV-1 Protease</B>
<br>--- Miguel L. Teodoro, George N. Phillips Jr., Lydia E. Kavraki
<br>[ <A HREF="pdf/98.pdf">PDF</A> ] [ <A HREF="ps/98.ps.gz">PS</A> ]

<P><li>
<B>Modelling of Enantioselectivity in <I>Candida antarctica</I> Lipase B</B>
<br>--- Sami Raza, Linda Fransson, Karl Hult
<br>[ <A HREF="pdf/99.pdf">PDF</A> ] [ <A HREF="ps/99.ps.gz">PS</A> ]

<P><li>
<B>Molecular Dynamics Study of the N-terminal Domain of Apolipoprotein E on a Mimetic-lipid Surface</B>
<br>--- M. Prevost
<br>[ <A HREF="pdf/100.pdf">PDF</A> ] [ <A HREF="ps/100.ps.gz">PS</A> ]

<P><li>
<B>Modeling of Chimeric <font face=symbol>&#98;</font>-galactosidase Antigenic Fusion Proteins</B>
<br>--- N. Boutonnet, M.Prevost
<br>[ <A HREF="pdf/101.pdf">PDF</A> ] [ <A HREF="ps/101.ps.gz">PS</A> ]

<P><li>
<B>Semianalytical Integration Method for Macromolecular Simulations</B>
<br>--- Dusanka Janezic, Matej Praprotnik
<br>[ <A HREF="pdf/102.pdf">PDF</A> ] [ <A HREF="ps/102.ps.gz">PS</A> ]

<P><li>
<B>On Negative Selection Against ATG Triplets Near Start Codons in Archaebactreial Genomes</B>
<br>--- Rintaro Saito, Masaru Tomita
<br>[ <A HREF="pdf/103.pdf">PDF</A> ] [ <A HREF="ps/103.ps.gz">PS</A> ]

<P><li>
<B>Computer Simulation of Human Erythorocyte and its Application to Analyses of Enzyme Deficiencies</B>
<br>--- Yoichi Nakayama, Ryo Matsushima, Ayako Kinoshita, Hironori Tanaka, Natsumi Noguchi, Masaru Tomita
<br>[ <A HREF="pdf/104.pdf">PDF</A> ] [ <A HREF="ps/104.ps.gz">PS</A> ]

<P><li>
<B>Modeling a Complex Gene Regulation Network Using the E-CELL System</B>
<br>--- Kenta Hashimoto, Sae Seno, Fumihiko Miyoshi, Masaru Tomita
<br>[ <A HREF="pdf/105.pdf">PDF</A> ] [ <A HREF="ps/105.ps.gz">PS</A> ]

<P><li>
<B>E-Cell: Software Environment for Whole Cell Simulation and Analysis</B>
<br>--- Kouichi Takahashi, Yusuke Saito, Tomohiko Aikawa, Tomoharu Iwata, Motohiro Yoneda, Masayuki Okayama, Masaru Tomita
<br>[ <A HREF="pdf/106.pdf">PDF</A> ] [ <A HREF="ps/106.ps.gz">PS</A> ]

<P><li>
<B>Comparative Study of Overlapping Genes in Bacterial Genomes</B>
<br>--- Yoko Fukuda, Takanori Washio, Yoichi Nakayama, Masaru Tomita
<br>[ <A HREF="pdf/107.pdf">PDF</A> ] [ <A HREF="ps/107.ps.gz">PS</A> ]

<P><li>
<B>On Base-Pairing Potential Between 16S rRNA and 5’UTR in Archaebacterial Genomes</B>
<br>--- Yuko Osada, Rintaro Saito, Masaru Tomita
<br>[ <A HREF="pdf/108.pdf">PDF</A> ] [ <A HREF="ps/108.ps.gz">PS</A> ]

<P><li>
<B>Stem Loop Structures in the <I>B.subtilis</I> Genome and Their Roles in the Transcription Machinery</B>
<br>--- Takanori Washio, Keio Asai, Kazuo Kobayashi, Naotake Ogasawara, Hiroki Yamamoto, Jun-ichi Sekiguchi, Ken-ichi Yoshida, Yasutaro Fujita, Masaru Tomita
<br>[ <A HREF="pdf/109.pdf">PDF</A> ] [ <A HREF="ps/109.ps.gz">PS</A> ]

<P><li>
<B>Comparative Analysis of Location of First Intron</B>
<br>--- Atsushi Sakurai, Shigeo Fujimori, Hiromi Kochiwa, Takanori Washio, Masaru Tomita
<br>[ <A HREF="pdf/110.pdf">PDF</A> ] [ <A HREF="ps/110.ps.gz">PS</A> ]

<P><li>
<B>Integrative Modeling of Mitochondrial Metabolism</B>
<br>--- Katsuyuki Yugi, Yuko Kanai, Masaru Tomita
<br>[ <A HREF="pdf/111.pdf">PDF</A> ] [ <A HREF="ps/111.ps.gz">PS</A> ]

<P><li>
<B>Analysis of CpG Dinucleotide Frequency in Bacterial Genomes with respect to Genomic Regions and Codon Position</B>
<br>--- Mami Goto, Takanori Washio, Masaru Tomita
<br>[ <A HREF="pdf/112.pdf">PDF</A> ] [ <A HREF="ps/112.ps.gz">PS</A> ]

<P><li>
<B>Analysis of Similarity Pattern and Selective Constraint in <I>C. elegans</I> and <I>C. briggsae</I> Genomes</B>
<br>--- Svetlana A. Shabalina, Alexey N. Spiridonov, Alexey S. Kondrashov
<br>[ <A HREF="pdf/113.pdf">PDF</A> ] [ <A HREF="ps/113.ps.gz">PS</A> ]

<P><li>
<B>A Reaction-Diffusion Model for Genetic Interference</B>
<br>--- Youhei Fujitani, Shintaro Mori, Ichizo Kobayashi
<br>[ <A HREF="pdf/114.pdf">PDF</A> ] [ <A HREF="ps/114.ps.gz">PS</A> ]

<P><li>
<B>Genomic Signature is Preserved in Short DNA Fragments</B>
<br>--- Patrik Deschavanne, Alain Giron, Joseph Vilain, Alexandra Vaury, Bernard Fertil
<br>[ <A HREF="pdf/115.pdf">PDF</A> ] [ <A HREF="ps/115.ps.gz">PS</A> ]

<P><li>
<B>Analysis of the Context of 5' Splice Site Sequences and 5' Splice Site-like Sequences in Mammalian mRNA Precursors by a Position Tree</B>
<br>--- Sumie K. Abe, Nobuyuki Takahashi, Mineichi Kudo, Masaru Shimbo, Akihiro Tsutsumi
<br>[ <A HREF="pdf/116.pdf">PDF</A> ] [ <A HREF="ps/116.ps.gz">PS</A> ]

<P><li>
<B>Insight on the Lateral Organization and the Size Distribution of the Gel and Fluid Clusters in DMPC/DSPC Lipid Bilayers</B>
<br>--- Ekaterina I. Michonova-Alexova, Istvan P. Sugar
<br>[ <A HREF="pdf/117.pdf">PDF</A> ] [ <A HREF="ps/117.ps.gz">PS</A> ]

<P><li>
<B>Investigation of Nucleosides Molecular Structure</B>
<br>--- S.A. Krasnokutski, A.Yu. Ivanov, G.G. Sheina, Yu.P. Blagoi
<br>[ <A HREF="pdf/118.pdf">PDF</A> ] [ <A HREF="ps/118.ps.gz">PS</A> ]

<P><li>
<B>Calculation of Thermodynamic Parameters for Helix-to-Coil Transition in Duplex and Triplex Complexes of Oligoadenilate</B>
<br>--- A. Shcherbakova, Yu. Blagoi, V. Zozulya
<br>[ <A HREF="pdf/119.pdf">PDF</A> ] [ <A HREF="ps/119.ps.gz">PS</A> ]

<P><li>
<B>Calculation of the Binding Isotherms of Cu<SUP>2+</SUP> and Ca<SUP>2+</SUP> Ions Interacting with DNA in Aqueous Solution</B>
<br>--- Elene V. Hackl, Natalya Bezlepkina, Yurij P. Blagoi
<br>[ <A HREF="pdf/120.pdf">PDF</A> ] [ <A HREF="ps/120.ps.gz">PS</A> ]



</OL>

</body>
</html>
